entity-framework/core/modeling/inheritance.md

```diff
@@ -8,28 +8,49 @@ uid: core/modeling/inheritance
 ---
 # Inheritance
 
-Inheritance in the EF model is used to control how inheritance in the entity classes is represented in the database.
+EF can map a .NET type hierarchy to a database. This allows you to write your .NET entities in code as usual, using base and derived types, and have EF seamlessly create the appropriate database schema, issue queries, etc. The actual details of how a type hierarchy is mapped are provider-dependent; this page describes inheritance support in the context of a relational database.
 
-## Conventions
+At the moment, EF Core only supports the table-per-hierarchy (TPH) pattern. TPH uses a single table to store the data for all types in the hierarchy, and a discriminator column is used to identify which type each row represents.
 
-By default, it is up to the database provider to determine how inheritance will be represented in the database. See [Inheritance (Relational Database)](relational/inheritance.md) for how this is handled with a relational database provider.
+> [!NOTE]
+> The table-per-type (TPT) and table-per-concrete-type (TPC), which are supported by EF6, are not yet supported by EF Core. TPT is a major feature planned for EF Core 5.0.
 
-EF will only setup inheritance if two or more inherited types are explicitly included in the model. EF will not scan for base or derived types that were not otherwise included in the model. You can include types in the model by exposing a *DbSet\<TEntity>* for each type in the inheritance hierarchy.
+## Included hierarchy types
 
-[!code-csharp[Main](../../../samples/core/Modeling/Conventions/InheritanceDbSets.cs?highlight=3-4&name=Model)]
+By convention, EF will only set up inheritance if two or more inherited types are explicitly included in the model. EF will not automatically scan for base or derived types that are not otherwise included in the model.
 
-If you don't want to expose a *DbSet\<TEntity>* for one or more entities in the hierarchy, you can use the Fluent API to ensure they are included in the model.
-And if you don't rely on conventions, you can specify the base type explicitly using `HasBaseType`.
+You can include types in the model by exposing a DbSet for each type in the inheritance hierarchy:
 
-[!code-csharp[Main](../../../samples/core/Modeling/Conventions/InheritanceModelBuilder.cs?highlight=7&name=Context)]
+[!code-csharp[Main](../../../samples/core/Modeling/Conventions/InheritanceDbSets.cs?name=InheritanceDbSets&highlight=3-4)]
 
-> [!NOTE]
-> You can use `.HasBaseType((Type)null)` to remove an entity type from the hierarchy.
+This model be mapped to the following database schema (note the implicitly-created *Discriminator* column, which identifies which type of *Blog* is stored in each row):
+
+![image](_static/inheritance-tph-data.png)
+
+>[!NOTE]
+> Database columns are automatically made nullable as necessary when using TPH mapping. For example, the *RssUrl* column is nullable because regular *Blog* instances do not have that property.
```

Comment:
Is this worth talking about this? Seems like an under-the-hood implementation detail, no?

Reply:
No preference but seen quite a few customers looking at migration and complaining that column is nullable even though they set it as Required.

Reply:
OK, if this is a frequent source of confusion we can leave it in.